symposion/cms/views.py

```diff
@@ -92,11 +92,10 @@ def file_create(request):
     if request.method == "POST":
         form = FileUploadForm(request.POST, request.FILES)
         if form.is_valid():
-            with transaction.commit_on_success():
```

Comment:
I'm not sure if this transaction was important. Is it intentional to remove this rather than replace with transaction.atomic() ?